docs/requirement-set-tables/powerpoint-1_1.md

```diff
@@ -0,0 +1,2 @@
+| Class | Fields | Description |
+|:---|:---|:---|
```

Comment:
FYI, the whats-new tool doesn't work on namespace-level functions, which is why this is blank. I thought about fixing it, but it's a lot of effort for this single scenario.